include/ReadAnyGroup.h

```diff
@@ -155,7 +155,8 @@ namespace ChimeraTK {
     void add(TransferElementAbstractor& element);
 
     ReadAnyGroup(const ReadAnyGroup&) = delete;
-    ReadAnyGroup(ReadAnyGroup&& other) noexcept;
+    ReadAnyGroup(ReadAnyGroup&& other) noexcept { operator=(std::move(other)); }
+    ReadAnyGroup& operator=(ReadAnyGroup&& other) noexcept;
```

Comment:
The commit "fixup: add ReadAnyGroup move assignment" probably should be squashed, too? (slightly misformatted, so autosquash wouldn't get it)

Reply:
yes. will force push squashed version now.